src/components/TransferTable.vue

```diff
@@ -149,7 +157,11 @@ export default {
                                 valueDisplay = 'Unknown precision';
                             }
                         } else {
-                            valueDisplay = `Id #${parseInt(log.topics[3], 16)}`;
+                            let tokenId = (this.tokenType === 'erc1155') ? BigNumber.from(log.data.substr(0, 66)).toString() : BigNumber.from(log.topics[3]).toString();
```

Comment:
'magic numbers' that might benefit from being a descriptive const variable for readability